design/m1_push_to_file_design.md

```diff
@@ -161,13 +161,13 @@ format, for which there will be one secrets file per Conjur secret.
 
 ##### Supported Annotations
 
-| Annotation                                     | Description |
-|------------------------------------------------|-------------|
-| conjur.org/conjur-secrets.{secret-group}       | List of secrets to be retrieved from Conjur. Each entry can be either:<ul><li>A Conjur variable path</li><li> A key/value pairs of the form `<alias>:<Conjur variable path>` where the `alias` represents the name of the secret to be written to the secrets file |
-| conjur.org/conjur-secrets-policy-path.{secret-group}  | Prefix to use for all Conjur variable paths for a secret group. An example is `prod/backend/`. The path is assumed to be relative to root policy path. A leading `/` is ignored. A trailing `/` is optional. Defaults to the root policy path. |
+| Annotation                                     | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
```

Comment:
We generally to not update design files after the development effort for that design is finished. Updating the documentation in `PUSH_TO_FILE.md` is great! Updating this design suggests that `properties` was in the original design, when it was not.

Reply:
Couldn't agree more 

Missed the fact that this is design up front. Just searched for all occurences in markdown. 😉